src/Item/Item.service.ts

```diff
@@ -592,6 +592,13 @@ export class ItemService {
       : null
 
     const upsertedItem: ItemAttributes = await Item.upsert(attributes)
+    if (dbItem && attributes.local_content_hash) {
+      // Update the Item Curation content_hash
+      await ItemCuration.update(
+        { content_hash: attributes.local_content_hash },
+        { item_id: item.id }
```

Comment:
Would you mind using attributes here?
```suggestion
        { item_id: attributes.id }
```